.github/workflows/push_issues_to_ccos.yml

```diff
@@ -0,0 +1,29 @@
+name: Push issues to CC Open Source
+on:
+  schedule:
+  - cron: '0 * * * *'
```

Comment:
I recommend doing this at a random time so that not everything runs at the same time.

Reply:
Resolved in 3e2e641.